manifests/init.pp

```diff
@@ -299,14 +313,17 @@
   Boolean $package_crmsh                                             = $corosync::params::package_crmsh,
   Boolean $package_pacemaker                                         = $corosync::params::package_pacemaker,
   Boolean $package_pcs                                               = $corosync::params::package_pcs,
+  Boolean $package_fence_agents                                      = $corosync::params::package_fence_agents,
   Optional[Array] $packageopts_corosync                              = $corosync::params::package_install_options,
   Optional[Array] $packageopts_pacemaker                             = $corosync::params::package_install_options,
   Optional[Array] $packageopts_crmsh                                 = $corosync::params::package_install_options,
   Optional[Array] $packageopts_pcs                                   = $corosync::params::package_install_options,
+  Optional[Array] $packageopts_fence_agents                          = $corosync::params::package_install_options,
```

Comment:
```suggestion
  Array[String[1]] $packageopts_fence_agen                           = $corosync::params::package_install_options,
```

This should work as well. It allows an empty array. If data is provided, it has to be strings with at least one character.